<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>571</width>
    <height>395</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="1" column="1">
     <widget class="QSlider" name="bSlider">
      <property name="maximum">
       <number>255</number>
      </property>
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>0</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="1" column="2">
     <widget class="QSlider" name="rSlider">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Minimum">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="maximum">
       <number>255</number>
      </property>
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
     </widget>
    </item>
    <item row="3" column="2">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>0</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>0</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QSlider" name="gSlider">
      <property name="maximum">
       <number>255</number>
      </property>
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>571</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>rSlider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>MainWindow</receiver>
   <slot>setColor()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>170</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>503</x>
     <y>498</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>gSlider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>MainWindow</receiver>
   <slot>setColor()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>331</x>
     <y>187</y>
    </hint>
    <hint type="destinationlabel">
     <x>319</x>
     <y>393</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>bSlider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>MainWindow</receiver>
   <slot>setColor()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>506</x>
     <y>232</y>
    </hint>
    <hint type="destinationlabel">
     <x>445</x>
     <y>452</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>rSlider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>label</receiver>
   <slot>setNum(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>157</x>
     <y>295</y>
    </hint>
    <hint type="destinationlabel">
     <x>160</x>
     <y>333</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>gSlider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>label_2</receiver>
   <slot>setNum(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>323</x>
     <y>291</y>
    </hint>
    <hint type="destinationlabel">
     <x>322</x>
     <y>333</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>bSlider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>label_3</receiver>
   <slot>setNum(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>491</x>
     <y>299</y>
    </hint>
    <hint type="destinationlabel">
     <x>495</x>
     <y>329</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>setColor()</slot>
 </slots>
</ui>
